resources/views/partials/components_form.blade.php

```diff
@@ -13,7 +13,7 @@
                 </li>
                 <div class="form-group group-items {{ $componentGroup->has_subscriber($subscriptions) ? null : "hide" }}">
                     @foreach($componentGroup->enabled_components()->orderBy('order')->get() as $component)
-                        @include('partials.component_input', compact($component))
+@include('partials.component_input', compact($component))
```

Comment:
Not that far left, sorry. Just so that it's directly underneath the `@foreach` or `@if` etc.